flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -73,17 +73,51 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
 
     private func loadContexts(result: @escaping FlutterResult) {
         SentrySDK.configureScope { scope in
+            // scope.setTag(value: "test", key: "test")
+            // scope.setExtra(value: "test", key: "test")
+            // scope.setUser(User(userId: "test"))
+            // scope.setDist("test")
+            // scope.setFingerprint(["test"])
+            // scope.setLevel(SentryLevel.warning)
+            // scope.add(Breadcrumb(level: SentryLevel.info, category: "test"))
+            // scope.setEnvironment("test")
+
             let serializedScope = scope.serialize()
             let context = serializedScope["context"]
 
             var infos = ["contexts": context]
 
+            if let tags = serializedScope["tags"] as? [String: String] {
+                infos["tags"] = tags
+            }
+            if let extra = serializedScope["extra"] as? [String: Any] {
+                infos["extra"] = extra
+            }
+            if let user = serializedScope["user"] as? [String: Any] {
+                infos["user"] = user
+            }
```

Comment:
@brustolin the `user` here isn't set, so where should I read the installation id from sentry-cocoa?